mods/ra/chrome/assetbrowser.yaml

```diff
@@ -1,5 +1,6 @@
 Background@ASSETBROWSER_PANEL:
 	Logic: AssetBrowserLogic
+		SupportedFormats: .shp, .r8, tmp, .tem, .des, .sno, .int, .jun, .vqa
```

Comment:
Dammit, I mixed the two lines in the diff view. >_> This is where I intended to mention that thirdparty mods might want to use their own extensions for the Asset Browser too. >_>